browser/src/VariantPage/VariantSiteQualityMetrics.tsx

```diff
@@ -571,7 +571,10 @@ const yTickFormat = (n: any) => {
   return `${n}`
 }
 
-const formatMetricValue = (value: any, metric: any) => {
+const formatMetricValue = (value: any, metric: any, isLog?: boolean) => {
+  if (isLog) {
+    return `log(${value})`
```

Comment:
naive question, is the default log here log10? that is the log we use in production (see slack [thread](https://atgu.slack.com/archives/CRA2TKTV0/p1725041402573139?thread_ts=1725037232.215369&cid=CRA2TKTV0))

Reply:
good catch! I updated to use log base 10, by default it was log base e.